src/scripts/servers.js

```diff
@@ -67,8 +67,14 @@ class Servers extends EventEmitter {
         if (Object.keys(hosts).length === 0) {
             const serverFileName = 'servers.json';
             const userDataDir = jetpack.cwd(remote.app.getPath('userData'));
+            const installDataDir = jetpack.cwd(process.cwd());
             try {
-                const result = userDataDir.read(serverFileName, 'json');
+                var result = installDataDir.read(serverFileName, 'json');
```

Comment:
@localguru So it doesn't have to read both files if the file exists in the user dir you could do

```
const result = userDataDir.read(serverFileName, 'json') || installDataDir.read(serverFileName, 'json');
```